indexer/database/bridge_transfers.go

```diff
@@ -246,14 +248,36 @@ func (db *bridgeTransfersDB) L2BridgeWithdrawal(txWithdrawalHash common.Hash) (*
 	return &withdrawal, nil
 }
 
-func (db *bridgeTransfersDB) L2BridgeWithdrawalSum() (float64, error) {
+func (db *bridgeTransfersDB) L2BridgeWithdrawalSum(filter models.WithdrawFilter) (float64, error) {
+	// Determine where filter
+	var clause string
+	switch filter {
+	case models.All:
+		clause = ""
+
+	case models.Finalized:
+		clause = "finalized_l1_event_guid IS NOT NULL"
+
+	case models.Proven:
+		clause = "proven_l1_event_guid IS NOT NULL"
+
+	default:
+		return 0, fmt.Errorf("unknown filter argument: %d", filter)
+	}
+
+	// NOTE - Scanning to float64 reduces precision versus scanning to big.Int since amount is a uint256
+	// This is ok though given all bridges will never exceed max float64 (10^308 || 1.7E+308) in wei value locked
+	// since that would require 10^308 / 10^18 = 10^290 ETH locked in the bridge
 	var sum float64
-	result := db.gorm.Model(&L2TransactionWithdrawal{}).Select("sum(amount)").Scan(&sum)
-	if result.Error != nil {
+	result := db.gorm.Model(&L2TransactionWithdrawal{}).Where(clause).Select("sum(amount)").Scan(&sum)
```

Comment:
```suggestion
	result := db.gorm.Model(&L2TransactionWithdrawal{}).Where(clause).Select("SUM(amount)").Scan(&sum)
```

Jus trying to be constent with capitalizing sql keywords